src/DIRAC/ConfigurationSystem/Client/Helpers/CSGlobals.py

```diff
@@ -22,7 +18,7 @@ def __load(self):
             return
         for extName in extensionsByPriority():
             try:
-                res = imp.find_module(extName)
+                res = importlib.import_module(extName)
                 if res[0]:
                     res[0].close()
                 self.__orderedExtNames.append(extName)
```

Comment:
I have few opinions on this:
 - Is `DIRAC.Core.Utilities.Extensions` about the same that `Extensions` class? maybe we should use one of them?
  - it is necessary to be defined that we want to receive here, for example it is not absolutely clear to me that should return `getExtensionData`.
 
 If we have enough "module spec" information, we can use `from importlib.util import find_spec`.
 
 ```
 importlib.import_module('DIRAC').__spec__ == find_spec('DIRAC')
 ```
 
 then can be simplified, for example:
 
 ```
   def __load(self):
        if not self.__modules:
            self.__modules = dict((ext, spec) for ext in extensionsByPriority() if (spec := find_spec(ext)))
 ```
 
 or if we always pass the name of the extension:
 
 ```
   def getExtensionPath(self, extName):
        return spec.submodule_search_locations[0] if (spec := find_spec(extName)) else None

    def getExtensionData(self, extName):
        return find_spec(extName)
 ```

Reply:
I think this `Extensions` class can just be removed entirely. For the two fuctions you mention:

* `getExtensionData` is unused
* There is only on use of `getExtensionPath` (which is actually pointless as the result is unused)